src/seedu/addressbook/AddressBook.java

```diff
@@ -711,15 +724,23 @@ private static void createFileIfMissing(String filePath) {
         }
 
         showToUser(String.format(MESSAGE_ERROR_MISSING_STORAGE_FILE, filePath));
+        createStorageFile(filePath, storageFile);
+    }
 
+	/**
+	 * Creates Storage File
+	 * @param filePath
+	 * @param storageFile
+	 */
+	private static void createStorageFile(String filePath, final File storageFile) {
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.